### ISSUE TITLE
rnn-predict-ensemble

### ISSUE THREAD

**Issue by williamteles** [dpinlab/mlex#21]
Create a predict to when using the dynamic sampler, the model predicts for every different sequence lenght individually and after average the logits to after apply the sigmoid
